### PR TITLE
Update BaseMenu.tsx

### PR DESCRIPTION
**Pull request by liujiayii**

### CODE REVIEW

File: src/SiderMenu/BaseMenu.tsx [hunk 1]
```diff
@@ -193,7 +193,7 @@ export default defineComponent({
       >
         {props.menuData &&
           props.menuData.map(menu => {
-            if (menu.meta.hidden) {
+            if (menu.meta?.hidden) {
```
Comment (sendya) [inline src/SiderMenu/BaseMenu.tsx:196]: It does not make sense

**Comment by sendya** [comment]
menu?.meta ？

**Comment by liujiayii** [comment]
也许是 menu?.meta?.hidden? 我可能传入了不太标准的菜单数组，报了hidden的异常

**Comment by sendya** [comment]
`BaseMenu` refactor render functional, this PR has expired

Thank your PR.

**Pull request closed without merging**